Rocket.Chat/Controllers/Auth/AuthViewController.swift

```diff
@@ -216,6 +216,11 @@ final class AuthViewController: BaseViewController {
                         strongSelf.performSegue(withIdentifier: "RequestUsername", sender: nil)
                     }
                 }
+            } else {
```

Comment:
@RocketChat/ios I just added this else block to avoid endless loadings if something goes wrong when retrieving the user object after fetching `MeRequest`. What do you guys think about it?

Reply:
👍